rules/helm/delete.tmpl.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
```

Comment:
Is this really better as a ruby script? Seems like shell (or even, `sh_binary`) might be good enough here.

Reply:
I wrote using Ruby for consistency with the other rule templates (even though there is still one shell script under these rules).